OpenGL.Platform/Input.cs

```diff
@@ -233,6 +259,22 @@ public static void AddKey(char key, bool shift, bool ctrl, bool alt)
             lock (keys) if (!keys.Contains(mkey)) keys.Add(mkey);
         }
 
+        /// <summary>
+        /// Adds a raw key that has been pressed to the list of keys that are currently held down.
+        /// </summary>
+        /// <param name="key">The key that was pressed.</param>
+        public static void AddKeyRaw(SDL.SDL_Scancode key)
+        {
+            char keyRaw = sdlKeyMap[key];
+            if(KeyBindingsRaw[keyRaw] != null && KeyBindingsRaw[keyRaw].Call != null)    // keybindings performs a lock of subqueue, ensuring thread safety
+            {
+                KeyBindingsRaw[keyRaw].Call(keyRaw, true);     // event is called immediately
```

Comment:
You're right, it makes no sense to subscribe an action with `char` param to an event with `SDL_Scancode`. Fixed it.
https://github.com/giawa/opengl4csharp/pull/42/commits/5ef201b70be4f5397d1690af17ebfd9cb7ccd8e7

Reply:
Key events are now stored in `Dictionary<SDL.SDL_Scancode, Event>` instead of `List<Event>`. Removed `sdlKeyMap`.
https://github.com/giawa/opengl4csharp/pull/42/commits/ca749e91fc6d9e79e9bf9133ad3307e7902dc85f